bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -1,195 +1,432 @@
+        Switch item="FaultIndication" icon="switch" label="Fault Indication"
+        Switch item="VentilationMode" icon="switch" label="Ventilation Mode"
+        Switch item="BypassStatus" icon="switch" label="Bypass Status"
+        Text item="BypassAutomaticStatus" label="Bypass Automatic Status"
+        Switch item="FreeVentilationStatus" icon="switch" label="Free Ventilation Status"
+        Switch item="FilterCheck" icon="switch" label="Filter Check"
+        Switch item="DiagnosticIndication" icon="switch" label="Diagnostic Indication"
+        Text item="ControlSetpoint" label="Control Setpoint"
+        Switch item="ServiceRequest" icon="switch" label="Service Request"
+        Switch item="ExhaustFanFault" icon="switch" label="Exhaust Fan Fault"
+        Switch item="InletFanFault" icon="switch" label="Inlet Fan Fault"
+        Switch item="FrostProtection" icon="switch" label="Frost Protection"
+        Text item="FaultCode" label="Fault Code"
+        Text item="DiagnosticCode" label="Diagnostic Code"
+        Text item="SystemType" label="System Type"
+        Switch item="Bypass" icon="switch" label="Bypass"
+        Text item="SpeedControl" label="Speed Control"
+        Text item="MemberID" label="Member ID"
+        Text item="OpenThermVersion" label="OpenTherm Version"
+        Text item="VersionType" label="Version Type"
+        Text item="RelativeVentilation" label="Relative Ventilation [%d %%]"
+        Text item="RelativeHumidity" label="Relative Humidity [%d %%]"
+        Text item="CO2Level" label="CO2 Level [%d ppm]"
+        Text item="SupplyInletTemperature" label="Supply Inlet Temperature [%.1f %unit%]"
+        Text item="SupplyOutletTemperature" label="Supply Outlet Temperature [%.1f %unit%]"
+        Text item="ExhaustInletTemperature" label="Exhaust Inlet Temperature [%.1f %unit%]"
+        Text item="ExhaustOutletTemperature" label="Exhaust Outlet Temperature [%.1f %unit%]"
+        Text item="ActualExhaustFanSpeed" label="Actual Exhaust Fan Speed [%d rpm]"
+        Text item="ActualInletFanSpeed" label="Actual Inlet Fan Speed [%d rpm]"
+        Switch item="NominalVentilationValueTransfer" icon="switch" label="Nominal Ventilation Value Transfer"
+        Text item="NominalVentilationValue" label="Nominal Ventilation Value"
+        Text item="NominalVentilationValue" label="Nominal Ventilation Value [%d %%]"
+        Text item="VentilationSetpoint" label="Ventilation Setpoint" minValue="0" maxValue="100" step="1"
+        Text item="TransparentSlaveParameterNumber" label="Transparent Slave Parameter Number"
+        Text item="TransparentSlaveParameterEntry" label="Transparent Slave Parameter Entry"
+        Text item="FaultHistoryBufferNumber" label="Fault History Buffer Number"
+        Text item="FaultHistoryBufferEntry" label="Fault History Buffer Entry"
+    }
+}
+```
+
+## Transparent Slave Parameters and Fault History Buffer channels
```

Comment:
These chapters should reside above the "Full Example" section.

Reply:
Please see my previous comment about readme structure.

Reply:
Done